File: Lib/Telerik/Examples/RssReader/RSS Library/RssChannel.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml.Serialization;

namespace RssReader.RSS_Library
{
    [XmlType("channel")]
    public class RssChannel
    {
        public RssChannel()
        {

        }

        [XmlElement("title")]
        public string Title { get; set; }

        [XmlElement("description")]
        public string Description { get; set; }

        [XmlElement("link")]
        public string Link { get; set; }

        [XmlElement("pubDate")]
        public string PubDate { get; set; }

        [XmlElement("item")]
        public RssItem[] Items { get; set; }

        [XmlElement("image")]
        public RssChannelImage Image { get; set; }
    }
}

File: Lib/Telerik/Examples/RssReader/RSS Library/RssChannel.vb
Imports System.Text
Imports System.Xml.Serialization
Imports System.Web.UI.WebControls

Namespace CS.RSS_Library
	<XmlType("channel")>
	Public Class RssChannel
		Public Sub New()

		End Sub

		<XmlElement("title")>
		Public Property Title() As String

		<XmlElement("description")>
		Public Property Description() As String

		<XmlElement("link")>
		Public Property Link() As String

		<XmlElement("pubDate")>
		Public Property PubDate() As String

		<XmlElement("item")>
		Public Property Items() As RssItem()

		<XmlElement("image")>
		Public Property Image() As RssChannelImage
	End Class
End Namespace
